js/form-data.js

```diff
@@ -0,0 +1,32 @@
+const formReloads = document.querySelector('#form');
+
+let formData = {
+  name: '',
+  email: '',
+  msg: '',
+};
+
+function formInputListens(inputType) {
+  formReloads.elements[inputType].addEventListener('input', () => {
+    formData[inputType] = formReloads.elements[inputType].value;
+    localStorage.setItem('form-data', JSON.stringify(formData));
+  });
+}
+
+function setFormInput(inputType) {
+  formReloads.elements[inputType].value = JSON.parse(localStorage['form-data'])[inputType];
+}
+
```

Comment:
- [ ] Kindly make sure to write code for `when the user loads the page, any saved data in the local storage have input fields with pre-filled data`. 
I have attached a video where when we refresh the page the data on the input files should be persistent. You have to use `window.onload ` for this. Kindly work on it.

https://user-images.githubusercontent.com/70269507/162329251-7857e17a-5b37-4013-a34f-855d14b550f2.mov

Reply:
Thank you so much for your suggestion. @sja-thedude